src/org/traccar/Context.java

```diff
@@ -258,7 +258,8 @@ public static TripsConfig initTripsConfig() {
                 config.getLong("report.trip.minimalTripDuration", 300) * 1000,
                 config.getLong("report.trip.minimalParkingDuration", 300) * 1000,
                 config.getLong("report.trip.minimalNoDataDuration", 3600) * 1000,
-                config.getBoolean("report.trip.useIgnition"));
+                config.getBoolean("report.trip.useIgnition"),
+                config.getBoolean("event.motion.processInvalidPositions"));
```

Comment:
Maybe we should rename it to `report.trip.` to be consistent with other parameters?

Reply:
I'm not sure. Firstly we have another parameter started from "event.motion" - `"event.motion.speedThreshold`, secondly frankly it is not working with trips, just with positions received online.

Reply:
Can we move `event.motion.speedThreshold` to the `TripsConfig` then?

Reply:
Sure. It will take a time.